src/main/java/frc/robot/commands/elevator/ElevatorToLimitSwitchCommand.java

```diff
@@ -0,0 +1,31 @@
+package frc.robot.commands.elevator;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.elevator.ElevatorSubsystem;
+
+/** Sends the elevator to position zero. */
+public class ElevatorToLimitSwitchCommand extends Command {
+    private ElevatorSubsystem elevatorSubsystem;
+    
+    /** Constructs a {@link ElevatorToLimitSwitchCommand} for the specified elevator. */
+    public ElevatorToLimitSwitchCommand(ElevatorSubsystem elevatorSubsystem) {
+        this.addRequirements(elevatorSubsystem);
+        this.elevatorSubsystem = elevatorSubsystem;
+    }
+
+    @Override
+    public void initialize() {
+        elevatorSubsystem.setMotorPower(0.2);
```

Comment:
Should probably be -.2?

Reply:
I think someone told me that the elevator motor's inversed, which means to go down u need positive power.

Reply:
Changed this to -0.2 for now.